dev-util/web_page_replay_go/web_page_replay_go-20220314.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit go-module
+
+EGO_SUM=(
+	"github.com/BurntSushi/toml v0.3.1/go.mod"
+	"github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d"
+	"github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d/go.mod"
+	"github.com/kylelemons/godebug v1.1.0"
+	"github.com/kylelemons/godebug v1.1.0/go.mod"
+	"github.com/pmezard/go-difflib v1.0.0"
+	"github.com/pmezard/go-difflib v1.0.0/go.mod"
+	"github.com/russross/blackfriday/v2 v2.0.1"
+	"github.com/russross/blackfriday/v2 v2.0.1/go.mod"
+	"github.com/shurcooL/sanitized_anchor_name v1.0.0"
+	"github.com/shurcooL/sanitized_anchor_name v1.0.0/go.mod"
+	"github.com/urfave/cli/v2 v2.3.0"
+	"github.com/urfave/cli/v2 v2.3.0/go.mod"
+	"golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2/go.mod"
+	"golang.org/x/net v0.0.0-20200602114024-627f9648deb9"
+	"golang.org/x/net v0.0.0-20200602114024-627f9648deb9/go.mod"
+	"golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod"
+	"golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd/go.mod"
+	"golang.org/x/text v0.3.0"
+	"golang.org/x/text v0.3.0/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/yaml.v2 v2.2.3/go.mod"
+	)
+go-module_set_globals
```

Comment:
EGO_SUM is deprecated, I have created a vendor tarball here: https://dev.gentoo.org/~sultan/distfiles/dev-util/web_page_replay_go/web_page_replay_go-20220314-deps.tar.xz